### DISCUSSION TITLE
Certificado Digital AWS

### DISCUSSION THREAD

**Discussion by davidncuba** [TadaSoftware/PyNFe#221] [General]
Pessoal estou acompanhando a um bom tempo o projeto e estou tentando agora implementar no meu sistema.
O problema é localmente o pfx funciona e roda perfeitamente, já se eu salvo ele na aws s3 e uso alguma forma de trazer ele já da problema erro de certificado alguém já passou por isso ?

**Comment by davidncuba** [comment]
Consegui resolver, puxei da aws e salvei em local temp depois disso passei o path para abrir o certificado

**Reply by EduardoRBarbosa** [reply]
poderia passar como foi feito isto ? estou com dificuldades
